spec/inheritance_spec.cr

```diff
@@ -90,4 +107,86 @@ describe "Classes inheriting GObject::Object" do
     LibGObject.g_type_check_instance_is_a(obj, UserSubject.g_type).should eq(1)
     obj.string.should eq("hey")
   end
+
+  it "can set GObject properties" do
+    obj = UserObjectWithGProperties.new
+
+    out_string = LibGObject::Value.new
+    LibGObject.g_value_init(pointerof(out_string), GObject::TYPE_STRING)
+    LibGObject.g_value_set_string(pointerof(out_string), "test value")
+    LibGObject.g_object_set_property(obj, "str-ing", pointerof(out_string))
+    LibGObject.g_value_unset(pointerof(out_string))
```

Comment:
You can use [g_object_set](https://docs.gtk.org/gobject/method.Object.set.html) to set the property and avoid dealing with GValue at all, GICrystal currently uses it to set properties.
```suggestion
    LibGObject.g_object_set(obj, "str-ing", "test value", Pointer(Void).null)
```